.github/ISSUE_TEMPLATE/introduce-sgid-dataset.md

```diff
@@ -0,0 +1,76 @@
+---
+name: Introduce data to the SGID
+about: Creation Checklist
+title: Add <dataset name> from <place>
+labels: 'introduction'
+assignees: ''
+---
+
+<!-- 
+Introduce your data, where it comes from, why it is being added etc
+-->
+# Summary
+
+We received yada yada from blahbidy blah. Foo Bar is the data contact and it will be updated quarterly. I propose we call it Baz and put it in this ISO category or group.
+
+<!-- 
+When the champion from your team has completed the triage, check [x] the checkbox
+-->
+
+# Triage
+
+- [ ] Data Team Triage Complete
+- [ ] Dev Team Triage Complete (assigned to)
+- [ ] Cadastre Team Triage Complete
+
+# We are introducing data
+
+The data was or will be added on `2020/00/00` to the following areas
+
+<!-- 
+add an [x] to the applicable areas you plan to add the data or ~strike~ out thoses that do not apply.
+If adding to the internal SGID with the hopes that swapper and forklift will push the item to SGID10 and AGOL
+you can strike them out. They are available if for some reason the internal sgid is skipped. 
+-->
+
+- [ ] Internal SGID
+- [ ] SGID10
+- [ ] ArcGIS Online
+
+<!--
+Where do we expect the data to show up. Check [x] all the areas 
+-->
+# The data should propagate to
+
+- [ ] ArcGIS Online
+- [ ] Internal SGID
+- [ ] SGID10
+- [ ] Open SGID
+- [ ] Open data
+- [ ] gis.utah.gov data pages
+- [ ] Stewardship doc and sgid-index
+- [ ] `SGID.META.AGOLItems`
```

Comment:
**question**: these three items have no automation so an addition to `We are introducing data` would not propagate to these place. Does anyone have a suggestion on a way to organize this? 

I think it should have the `(@assigned )` part of it so we know who is doing it or has done it.